bodhi/__init__.py

```diff
@@ -0,0 +1 @@
+__import__('pkg_resources').declare_namespace(__name__)
```

Comment:
I believe this is not needed - it was something I was trying when it wasn't working last night. I'm going to try dropping it and testing that things still work. If this isn't needed, I can also drop the ```namespace_packages=['bodhi']``` lines from the ```setup()``` calls.